patches/phing-relative-symlinks.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/classes/phing/tasks/ext/SymlinkTask.php b/classes/phing/tasks/ext/SymlinkTask.php
```

Comment:
Can we add a README to the patches directory that explains where this patch comes from?

Reply:
I went ahead and added a README.